lib/chainweb/util/network.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Kadena.Chainweb.Network do
```

Comment:
@santiagoboterokommit Are you sure this code will allow users to move between networks without changing environment variables?. Are you anticipating that?